cmd/zoekt-indexserver/fetch.go

```diff
@@ -65,7 +55,7 @@ func callGetReposModifiedSinceForCfgs(ctx context.Context, cfgs []ConfigEntry, l
 		cmd.Args = append(cmd.Args, createGithubArgsMirrorAndFetchArgs(ctx, c)...)
 		cmd.Args = append(cmd.Args, "-since", lookbackInterval.Format(iso8601Format))
 
-		stdout, _ := loggedRun(ctx, cmd)
+		stdout, _ := loggedRun(trace, cmd)
```

Comment:
❓ same `loggedRun` question, `repoDir` and `lookbackInterval` are both passed in cmd.Args.